apps/src/templates/sectionProgress/progressTables/ProgressTableSummaryCell.jsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import color from '@cdo/apps/util/color';
+
+const styles = {
+  container: {
+    height: 20,
+    margin: 10,
+    boxSizing: 'border-box',
+    borderWidth: 1,
+    borderStyle: 'solid'
+  }
+};
+
+export default class ProgressTableSummaryCell extends React.Component {
+  static propTypes = {
+    studentId: PropTypes.number.isRequired,
+    statusPercents: PropTypes.object,
+    assessmentStage: PropTypes.bool,
+    onSelectDetailView: PropTypes.func
+  };
+
+  heightForPercent(percent) {
+    return `${percent}%`;
+  }
+
+  render() {
+    const {statusPercents, assessmentStage} = this.props;
+    const started =
+      statusPercents.attempted > 0 || statusPercents.incomplete < 100;
```

Comment:
Just curious, why do we need both of these checks?

Reply:
So it looks like `statusPercents.incomplete`= `statusPercents.attempted` + `other incomplete statuses`. So you could have a lesson where `statusPercents.incomplete` = 100, but 2 levels have been attempted which would be considered "started". Or one level is completed and the rest haven't been started, then `statusPercents.attempted` = 0, and `statusPercents.incomplete` < 100 which would also be considered "started". Let me know if that makes sense, I can update the code with a comment.

Reply:
Maybe a better way to write this would be 
`const started = statusPercents.attempted + statusPercents.completed + statusPercents.imperfect > 0`

Reply:
After a chat with Charlie, we decided on a refactor of `statusPercents` which will resolve this confusion, changes coming..